### PR TITLE
140921999-Create-the-ability-to-segment-opportunities-by-lot

### PR DESCRIPTION
**Pull request by benvand**
Create the ability to segment opportunities by lot in GA

* Add slug data to template
* Look for the slug and report it to Google Analytics in JS

### CODE REVIEW

**Comment by idavidmcdonald** [comment]
where are the tests... ;)

File: spec/javascripts/unit/AnalyticsSpec.js [hunk 1]
```diff
@@ -293,4 +293,33 @@ describe("GOVUK.Analytics", function () {
     });
 
   });
+
+
+  describe("Opportunity page", function() {
+    var lotData = $('<span data-lot="test-lot"></span>');
+    var argsCB = function (i) {i.args};
```
Comment (idavidmcdonald) [inline spec/javascripts/unit/AnalyticsSpec.js]: Sorry, what does argsCB do?

File: setup.cfg [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [pep8]
-exclude = ./venv,./bower_components,./node_modules,./app/content
+exclude = ./venv*,./bower_components,./node_modules,./app/content
```
Comment (benvand) [inline setup.cfg]: Done

File: app/assets/javascripts/analytics/_pageViews.js [hunk 3]
```diff
@@ -73,6 +74,10 @@
           }
         }
       }
+      if (/^\/digital-outcomes-and-specialists(.*?)\/opportunities\/(\d+)$/.test(pathname)) {
```
Comment (benvand) [inline app/assets/javascripts/analytics/_pageViews.js]: Great shout, it was actually formatted completely differently. 
Especially important given the code style presentation im giving on Monday 😬

**Pull request merged**